### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by KUBAHO3**
In this project I have:

- added linter setup and fix all linter errors
- added web pack plugins and connected files
- added proper documentation of the project in the readme.md file

To run this project in your system please consider the following practices:

- [ ] clone this GitHub repository `git clone https://github.com/KUBAHO3/Leaderboard` and navigate to the Project-setup branch `cd Leaderboard`, `git checkout Project-setup`
- [ ] install all node packages by `npm install`
- [ ] run the following command in your terminal to build the system on your dev `npm run build`
- [ ] Navigate to the `base_url/dist/`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
```
Comment (wayungi) [inline package.json]: - [ OPTIONAL ] Kindly write a proper description for your title 👍🏾

File: README.md [hunk 2]
```diff
@@ -60,229 +61,184 @@ After you're finished please remove all the comments and instructions!
...
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+:bust_in_silhouette: **Yusuf Sholotan**
+- GitHub: [@yin-ka](https://github.com/yin-ka)
```
Comment (wayungi) [inline README.md]: - [ OPTIONAL ] This is an individual project but you have 2 authors here. Kindly fix this. 🤝  
authos

**Review by wayungi — APPROVED** [review]
Hi @KUBAHO3 , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [x] All linter checks passing ✔️ 
- [x]  Correct Gitflow  ✔️ 
- [x] Descriptive PR body ✔️ 


## Optional suggestions

- [x] Very good work writing commit messages in the imperative tense. Please remember to always capitalize the first word
commits

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (KUBAHO3) [reply package.json]: Noted as well

File: README.md (hunk 2 above)
Comment (KUBAHO3) [reply README.md]: updated to meet requirement

**Pull request merged**